.vscode/settings.json

```diff
@@ -0,0 +1,7 @@
+{
+    "typescript.tsc.autoDetect": "off",
+    "grunt.autoDetect": "off",
+    "jake.autoDetect": "off",
+    "gulp.autoDetect": "off",
+    "npm.autoDetect": "off"
```

Comment:
These are necessary so that VS Code will not automatically populate its _Tasks_ menu with auto-detected stuff (which won't anyway for Metabase front-end code).